tests/README.md

```diff
@@ -2,14 +2,13 @@
 
 Welcome to the Phalcon Testing Suite.
 
-This folder includes all the unit tests that test Phalcon components, ensuring that you enjoy a bug free framework.
+This folder contains all the tests for the Phalcon Framework. 
 
 ## Getting Started
 
-This testing suite uses [Travis CI][0] for each run. Every commit pushed to this repository will queue a build into the continuous integration service and will run all
-tests to ensure that everything is going well and the project is stable.
+This testing suite uses [Travis CI][0] for each run. Every commit pushed to this repository will queue a build into the continuous integration service and will run all tests to ensure that everything is going well and the project is stable.
 
-The testing suite can be run on your own machine. The main dependency is  [Codeception][1] which can be installed using [Composer][6]:
+The testing suite can be run on your own machine. The only dependencies for running the testing suite or even developing on Phalcon is [nanobox][9]`. Installation instructions for nanobox can be found [here][10]. Also, the main dependency is [Codeception][1] which can be installed using [Composer][6]:
```

Comment:
This is phrased a bit awkwardly... Maybe .... `The only dependencies for running the testing suite are [nanobox][9] and [Codeception][1]. Nanobox can also be used for developing Phalcon, and its installation instructions can be found [in the Nanobox documentation][10]. Codeception can be installed using [Composer][6]:` or something similar? (also, the `[nanobox][9]` here isn't rendering as a link...)